### DISCUSSION TITLE
RFC: Initial Discussion Around Topping Up the KTON Liquidity Farm for 6 More Months

### DISCUSSION THREAD

**Discussion by 2xJDubs** [ktondao/KtonDAO#31] [Q&A]
With the success we have seen with the current liquidity farming for the KTON/RING pool on Snowswap, I wanted to start the discussion on making a new request for the treasury to top up the farm funds for another 6 months.  To reminder everyone, here is the original discussion for developing the farm, https://github.com/orgs/darwinia-network/discussions/1414.

With the incentive the KTON/RING pool on Snowswap has attracted close to the 2000 KTON that was the original goal, currently there are 1639 KTON in the swap pool.

![image](https://github.com/user-attachments/assets/b9ffff96-81c8-4c8c-810b-edd57db1d164)

The farming pool will run out of rewards in 2 Months and 18 days which is why I am starting this discussion now as the voting for the treasury will take 30ish days on its own.

![image](https://github.com/user-attachments/assets/b8042bfe-a63c-4a12-8f4c-63d3256d3827)

The original amount requested for the farm was 1.4M RING and I think that would be a good amount to top up with for the next 6 month.  

So in the end I would like to get feedback on requesting 1,400,000 RING from the Darwinia Treasury to fund another 6 months of the KTON/RING farm.  

Happy to discuss making this amount lower as well if the majority thinks it doesn't need to be the same as before.

Thank you.

**Comment by SasoLithops** [comment]
Not to burden treasury as much I would propose halving of the rewards to 700k as bullrun is approaching and hopefully Ring and Kton can start making profit for farmers on their own. APR will still be huge.

Also liquidity is set close to goal so no need to attract more users.

Maybe we could do a retroactive airdrop of the remaining Pink tokens in DCDAO treasury to first half year LP providers. Those tokens still have some small value, best to airdrop them soon.
For next half year TNKR and VARCH could be airdropped.

**Comment by Laki389** [comment]
> So in the end I would like to get feedback on requesting 1,400,000 RING from the Darwinia Treasury to fund another 6 months of the KTON/RING farm.

In the past period, looking at the pool, it was constantly between 1650 - 1850 KTON, which was close to the set target of 2000 KTON.

The reason why there is not constantly over 2000 KTON is IMHO, because in the meantime a new staking website for KTON was made.
Even in Darwinia 2.0 there is unmigrated KTON so the staking rewards are bigger than we could have predicted.

But as the remaining non-migrated KTON slowly migrates, it is expected that the staking rewards will decrease, which would be enough for that 1.4M RING to attract over 2000 KTON to Snowswap.

To conclude, **I support 1.4 M RING for another 180 days.** :+1:

**Reply by cryptohustler101** [reply]
With the goal of reaching 2k KTON and market conditions stabilizing, the 1.4M RING will attract more users to Darwinia. This influx of KTON will draw long-term holders to deposit their KTON. I believe this initiative is crucial for sustained activity and will encourage others to build on Darwinia, capturing these new participants. Therefore, I vote yes on @2xJDubs' proposal for 1.4M RING, as it could bring in another 2k KTON, creating a solid LP for Snowswap and fostering organic ecosystem growth!

**Comment by hackfisher** [comment]
After the establishment and operation of Kton DAO, I lean towards having Kton DAO governance and allocating a small proportion of Kton DAO's own income for farm incentives (or buyback from DEX KTON/RING pair and add liquidity KTON/RING), rather than applying for additional expenditures from the Treasury. However, before this, due to the lack of reasonable means, I did not oppose appropriate Treasury proposals to continue farm incentives.

**Reply by 2xJDubs** [reply]
I absolutely agree that once KTONDAO is fully active it should take over supporting this initiative.  One of the reasons i wanted to keep this request to 6 months so we can again evaluate how things are going.  Thank you for your input.

**Comment by SasoLithops** [comment]
Can someone explain to me why at current Ring/Crab price daily incentives for farming in pools are:

For Ring/Kton is 21$

but 

for Crab/Ckton is just below 0.6$.

Anyone new(except for us hardcore belivers) checking $ price and not ammount of Ring tokens they get for risking and providing LP probably doesn't care much about huge APR and incentives at all. As $ value is too low. And with bigger investment the APR would of course go down.
Point I am trying to make, doubt we will get more Kton in pool with more burden from treasury as all we do is put more Ring on market. Imho those two pools above are pools mostly for hardcore belivers.

**Comment by hackfisher** [comment]
I would suggest to use and propose to https://www.tally.xyz/gov/ktondao to allocate part of the incomes for further liquidity support.

**Reply by Laki389** [reply]
> I would suggest to use and propose to https://www.tally.xyz/gov/ktondao to allocate part of the incomes for further liquidity support.

![Screenshot_3](https://github.com/user-attachments/assets/3f015451-103f-4174-9e80-713e688e4ca8)

The farm incentive for KTON-RING LP ends in **48 days.**

Is that enough time for KtonDAO to debate, propose solutions, find the best, and finally put it to a vote?
Even after it is voted it will take time for Kton DAO to raise its own funds.

**I am afraid that there will be some inter-period of several months in which there will be no incentive and the liquidity pool will be emptied.**

I think it is good for the last time  a proposal to be made for a donation from Darwinia Treasury to enable the Snowswap incentive.

A possible compromise would be to shorten the duration from 6 to 4 or 3 months.
Instead of an incentive of 1.4M Ring for 6 months, we could:

- 933 333 Ring for 4 months

or

- 700 000 Ring for 3 months

**Reply by hackfisher** [reply]
I think it is easier via KtonDAO, 48 days is not that short, and KtonDAO can directly continually reserve some RINGs from reward and buyback some KTON to add liquidity to DEX instead of do incentive indirectly(This saves RING spending and preferred by RingDAO too).

I can not find a reason RingDAO/Treasury should continue compensating KtonDAO as there is no further technical barrier exists any more.

**Reply by Laki389** [reply]
> I think it is easier via KtonDAO, 48 days is not that short, and KtonDAO can directly continually reserve some RINGs from reward and buyback some KTON to add liquidity to DEX instead of do incentive indirectly(This saves RING spending and preferred by RingDAO too).
> 
> I can not find a reason RingDAO/Treasury should continue compensating KtonDAO as there is no further technical barrier exists any more.

Yes, but it will take months to make, for example 1500+ KTON liquidity.

According to previous experience, those 48 days will not even be enough for discussion, proposals and formal voting.

Even when passed it will take months to make satisfactory liquidity.
KtonDAO currently has zero funds.

I thought, to make one last donation from the Darwinia Treasury.

But in case you insist, since the voting ratio is known where only Itering is able to provide the necessary Support,

then i guess it is futile to continue this discussion and Kton DAO should be let know to take care of this problem. :+1:

**Reply by hackfisher** [reply]
For easier calculation, here is the reward txs from Treasury to KtonDAOVault:

https://explorer.darwinia.network/address/0x652182C6aBc0bBE41b5702b05a26d109A405EAcA


Option 1,Each day around 80K RING, if we propose to allocate 20% of it, it will be 16K per day, using half aka. 8K RING to buy around 8 KTON, and add them as liquidity to DEX.

Alternatively, option2, add 16K in sum as incentive for liquidity farm, which is more compared to proposed incentive '700 000 Ring for 3 months'.

I personally prefer option 1 which is time and holder friendly.

**Reply by Laki389** [reply]
> For easier calculation, here is the reward txs from Treasury to KtonDAOVault:
> 
> https://explorer.darwinia.network/address/0x652182C6aBc0bBE41b5702b05a26d109A405EAcA
> 
> Option 1,Each day around 80K RING, if we propose to allocate 20% of it, it will be 16K per day, using half aka. 8K RING to buy around 8 KTON, and add them as liquidity to DEX.
> 
> Alternatively, option2, add 16K in sum as incentive for liquidity farm, which is more compared to proposed incentive '700 000 Ring for 3 months'.
> 
> I personally prefer option 1 which is time and holder friendly.

I think it's better  a new discussion to be opened for this.  :+1:

Also, where will the Kton DAO discussions take place?
Here on Darwinia Github?

**Reply by SasoLithops** [reply]
Here:

> https://github.com/darwinia-network/KtonDAO

**Reply by SasoLithops** [reply]
_I personally prefer option 1 which is time and holder friendly._

I would only use part of RING rewards for incentives not actually provide liquidity.
If it is used for actual liquidity KTON is taken off market and imediately put back in LP. Where it is at bigger risk and will be hurt by impermanent loss.
While buybacks would put it in treasury and off the market until KTON DAO decide what to do with it.

**Reply by Laki389** [reply]
> For easier calculation, here is the reward txs from Treasury to KtonDAOVault:
> 
> https://explorer.darwinia.network/address/0x652182C6aBc0bBE41b5702b05a26d109A405EAcA
> 
> Option 1,Each day around 80K RING, if we propose to allocate 20% of it, it will be 16K per day, using half aka. 8K RING to buy around 8 KTON, and add them as liquidity to DEX.
> 
> Alternatively, option2, add 16K in sum as incentive for liquidity farm, which is more compared to proposed incentive '700 000 Ring for 3 months'.
> 
> I personally prefer option 1 which is time and holder friendly.

@hackfisher What is your opinion on my, let's call it the **third option**?

https://darwinia.subscan.io/account/0x6d6f646c64612f74727372790000000000000000

Darwinia Treasury has **779 KTON.**

I think they should be donated to KtonDAO and used to build initial liquidity.

Let's say, for example, that 84 000 daily RING inflation is going to KTON DAO.
20% of that, or roughly 16k RING to go towards making KTON - RING LP until a pool of 2000 KTON and an appropriate number of RING is made.


**Calculation under an assumed ratio of 1 kton = 1000 ring.**

For 2000 kton and 2M RING we will lack 1221 KTON and 2M RING,

i.e. 3 221 000 RING.
**It will take about 200 days to collect this amount.**

After this, the overall daily inflation of around 84k RING should start going again for KTON staking, as it is now.

Also, I think some other DEX should be considered since Snowswap is outdated and inactive/no team.

There was some info/talks that HELIX will be doing DEX, how far along is that?
Maybe HELIX or some other option should be considered. :+1:

**Reply by SasoLithops** [reply]
I support idea of KTON being donated to KTON DAO as it was forgotten. 

But still I think we shouldn't burden KTON stakers and KTON DAO to provide liquidity from funds they get but should incentivise users to provide liquidity. We have seen that is working now. 
I think it would be great if Darwinia treasury will have to fund some more last time for incentives ( like Laki proposed it doesn't have to be 6 months if you find it too long) until KTON DAO has some time to set a mechanism to be self sustainable. If we would just drop the rewards and start building own liquidity, I  expect KTON sell pressure to be huge.As the one utility it has atm will be lowered for liquidity that already exists.

**Reply by hackfisher** [reply]
> I think it would be great if Darwinia treasury will have to fund some more last time for incentives ( like Laki proposed it doesn't have to be 6 months if you find it too long) until KTON DAO has some time to set a mechanism to be self sustainable. If we would just drop the rewards and start building own liquidity, I expect KTON sell pressure to be huge.As the one utility it has atm will be lowered for liquidity that already exists.

Explain why this is not a burden for Treasury/RingDAO?

**Reply by hackfisher** [reply]
> I support idea of KTON being donated to KTON DAO as it was forgotten.

Preferred to leave it to RING token holders and RingDAO to decide as the they are the owner of it and decide to donate or not, I have no idea on this for now.

**Reply by SasoLithops** [reply]
_Explain why this is not a burden for Treasury/RingDAO?_

At first I want to mention I was for halving the rewards to 700k but we did a poll in Darwinia Official TG. It was vote either for 1.4M or 700k. 1.4M won by 10:9 votes. 
Since majority of community voted for 1.4 M, I will use 1.4M RING as example.

Looking at current data on Coingecko( I will attach image below the post ) there was 40 000 USD daily trading (I took only data for MEXC and Uniswap, Coin W has 300 000 USD but don't know the exchange maybe just bots). But let's say some bots and some days higher some days lower, let's take 20 000USD daily trading volume on all exchanges. 
Asking for 1.4M Ring which will be spread over 6 months is at current price cca 2800 USD. So we are asking for small ammount. So if I divide that number by 180 days (half a year) it gives 16 USD per day. If we take into account that due to past RING price movement  most will sell. They probably sell daily and weekly. Let's  say weekly. 7 days times 16 USD gives 112 USD weekly sell pressure if everyone sells. I think market with 20 000 USD daily trading can handle that.


![Screenshot_2024-08-23-08-30-43-437_com.android.chrome.jpg](https://github.com/user-attachments/assets/e39c318e-8c55-4ee8-ac2f-9962b46e7fcd)

**Reply by SasoLithops** [reply]
_Preferred to leave it to RING token holders and RingDAO to decide as the they are the owner of it and decide to donate or not, I have no idea on this for now._

Absolutely agree. Not my decision to make, just expressing my view.

**Reply by Laki389** [reply]
> I support idea of KTON being donated to KTON DAO as it was forgotten.
> 
> But still I think we shouldn't burden KTON stakers and KTON DAO to provide liquidity from funds they get but should incentivise users to provide liquidity. We have seen that is working now. I think it would be great if Darwinia treasury will have to fund some more last time for incentives ( **like Laki proposed it doesn't have to be 6 months if you find it too long**) until KTON DAO has some time to set a mechanism to be self sustainable. If we would just drop the rewards and start building own liquidity, I expect KTON sell pressure to be huge.As the one utility it has atm will be lowered for liquidity that already exists.


I want to emphasize just in case, in case I have been misunderstood or misinterpreted,

I suggested a one-time, shorter period of, for example, 3 months,

because I think that the period of 43 days will not be enough for the so-called decentralized debate and governance, after which, logically, the liquidity of Snowswap will disappear.

With a one-time donation that will provide a 90-day incentive for KTON - RING LP farmers, enough time will be bought for the young and new KTON DAO, which with an empty treasury is trying to figure out what to do next.

**Reply by Laki389** [reply]
> > I support idea of KTON being donated to KTON DAO as it was forgotten.
> 
> Preferred to leave it to RING token holders and RingDAO to decide as the they are the owner of it and decide to donate or not, I have no idea on this for now.

There is another option, instead of Ring DAO donating its KTON, it can enable its KTON as liquidity.

Really worth thinking about, why would the new KTON DAO, with an empty treasury, be solely responsible for liquidity, when it is KTON-RING liquidity, both tokens equally have the only opportunity to trade on the Darwinia chain.

I plan to do a poll on this.

**Reply by SasoLithops** [reply]
You make great point. I agree both DAOs should equaly provide liquidity. Voted on that option.

But I would rather that they provide farming incentives not actual liquidity.

**Comment by johneggleston** [comment]
I am definitely in favour of halving the rewards to 700k Ring for the next cycle. Though there are a couple features that could be added to the Snowswap interface that I think would be cool ways to fuel activity, and incentivise liquidity provision and trading. 

Firstly, displaying the LP reward APR alongside the farm APR would be cool (showing people how much they can earn from the organic LP fees on top of the farm rewards if they provide liquidity). 

Secondly, displaying the price discrepancies between the Snowswap pool and pools on ETH/ARB pools to encourage trading activities from the constant arbitrage opportunities. 

Hoped you liked my input!
